packages/reports/tests/integration/components/print-report-view-test.js

```diff
@@ -1,131 +1,149 @@
 
-const RESPONSE = {
+const TEMPLATE = hbs`
+<PrintReportView
+  @report={{this.report}}
+  @response={{this.response}}
+/>`;
+
+const RESPONSE = NaviFactResponse.create({
   rows: [
     {
       adClicks: 1707077,
-      dateTime: '2015-11-09 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-09 00:00:00.000'
     },
     {
       adClicks: 1659538,
-      dateTime: '2015-11-09 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-09 00:00:00.000'
     },
     {
       adClicks: 1977070,
-      dateTime: '2015-11-11 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-11 00:00:00.000'
     },
     {
       adClicks: 1755382,
-      dateTime: '2015-11-12 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-12 00:00:00.000'
     },
     {
       adClicks: 1348750,
-      dateTime: '2015-11-13 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-13 00:00:00.000'
     },
     {
       adClicks: 856732,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     },
     {
       adClicks: 716731,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     },
     {
       adClicks: 399790,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     },
     {
       adClicks: 699490,
-      dateTime: '2015-11-14 00:00:00.000'
+      'network.dateTime(grain=day)': '2015-11-14 00:00:00.000'
     }
   ]
-};
+});
 
 module('Integration | Component | print report view', function(hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(async function() {
     const metadataService = this.owner.lookup('service:navi-metadata');
-    const store = this.owner.lookup('service:store');
 
     await metadataService.loadMetadata();
     this.set('response', RESPONSE);
 
     //set report object
-    this.set(
-      'report',
-      store.createRecord('report', {
-        request: store.createFragment('bard-request/request', {
-          logicalTable: store.createFragment('bard-request/fragments/logicalTable', {
-            table: metadataService.getById('table', 'spaceId', 'bardOne'),
-            timeGrain: 'day'
-          }),
-          responseFormat: 'csv',
-          intervals: A([{ interval: new Interval('current', 'next') }])
-        }),
-        visualization: {
-          type: 'line-chart',
-          version: 1,
-          metadata: {
-            axis: {
-              y: {
-                series: {
-                  type: 'metric',
-                  config: {
-                    metrics: [
-                      {
-                        metric: 'adClicks',
-                        parameters: {},
-                        canonicalName: 'adClicks',
-                        longName: 'Ad Clicks'
-                      }
-                    ]
-                  }
+    this.set('report', {
```

Comment:
Why is this no longer doing `store.createRecord`?

Reply:
When keeping the same format of just defining the request and visualization the render failed. I could be messing something up though.